src/Core/Import/EntityField/Provider/AddressFieldsProvider.php

```diff
@@ -56,9 +56,9 @@ public function getCollection()
         $fields = [
             new EntityField('id', $this->trans('ID', 'Admin.Global')),
             new EntityField('alias', $this->trans('Alias', 'Admin.Shopparameters.Feature'), '', true),
-            new EntityField('active', $this->trans('Active  (0/1)')),
-            new EntityField('customer_email', $this->trans('Customer email'), '', true),
-            new EntityField('id_customer', $this->trans('Customer ID')),
+            new EntityField('active', $this->trans('Active  (0/1)', 'Admin.Advparameters.Feature')),
```

Comment:
Is it really useful since the default domain of this class' `trans` method is `Admin.Advparameters.Feature` ?